jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetLambdaHandlerResolver.kt

```diff
@@ -10,10 +10,11 @@ import com.intellij.psi.NavigatablePsiElement
 import com.intellij.psi.PsiElement
 import com.intellij.psi.search.GlobalSearchScope
 import com.jetbrains.rd.framework.impl.RpcTimeouts
+import com.jetbrains.rd.framework.impl.startAndAdviseSuccess
```

Comment:
Is `DotNetLambdaHandlerResolver` really needed after moving to a backend line marker providing?